tools/plugin/scan-agent-plugins.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
@kezhenxu94 Please review this script.

@EvanLjp Tool script should be used in the CI process. Like the license check script. Check the result with the markdown document, make sure the new contributors have updated the doc.

Reply:
LGTM except for the code style

Reply:
> @kezhenxu94 Please review this script.
> 
> 
> 
> @EvanLjp Tool script should be used in the CI process. Like the license check script. Check the result with the markdown document, make sure the new contributors have updated the doc.

Refer to the check-dependencies.sh script, your scrip is expected to be rewrote to check actual plugin list and the md doc, instead of simply generating a list, of course you can print the diff so that one knows what's wrong when running your script, another tip is to put this check at the very beginning of the CI stages, because it's more lightweight and doesn't depends on the final tar (while check-dependencies.sh does)

Reply:
> @kezhenxu94 Please review this script.
> 
> 
> 
> @EvanLjp Tool script should be used in the CI process. Like the license check script. Check the result with the markdown document, make sure the new contributors have updated the doc.

Refer to the check-dependencies.sh script, your scrip is expected to be rewrote to check actual plugin list and the md doc, instead of simply generating a list, of course you can print the diff so that one knows what's wrong when running your script, another tip is to put this check at the very beginning of the CI stages, because it's more lightweight and doesn't depends on the final tar (while check-dependencies.sh does)

Reply:
push again, the script looks like check-LICENSE.sh